test/test_nestedtensor.py

```diff
@@ -7014,12 +7028,12 @@ def test_layout_conversion_backward(self, device):
     "jiterator_binary",
     "jiterator_binary_return_by_ref",
     "jiterator_unary",
-    # Bug found: sum() with keepdim=True returns invalid shape
-    "sum",
     # RuntimeError: prod(): keepdim=True must be set for NestedTensor
     "prod",
     # RuntimeError: "jagged_to_padded_dense" not implemented for 'Bool'
     "nanmean",
+    # AssertionError: assert not isinstance(out_ref_component, (list, tuple))
+    "native_layer_norm",
```

Comment:
Are there any particular inputs that is problematic? 
(if that is the case, maybe worth opening a new issue with some more details, and linking back here)

Reply:
Actually looking at CI, this is actually succeding :P